evalml/tests/data_checks_tests/test_datetime_format_data_check.py

```diff
@@ -349,35 +352,6 @@ def test_datetime_format_data_check_multiple_errors():
         get_uneven_error("dates", ww_payload),
     ]
 
-    dates = (
-        pd.date_range("2021-01-01", periods=15, freq="2D")
-        .drop("2021-01-13")
-        .append(pd.date_range("2021-01-30", periods=1))
-        .append(pd.date_range("2021-01-31", periods=86, freq="2D"))
-    )
-    X = pd.DataFrame({"dates": dates}, dtype="datetime64[ns]")
-
-    ww_payload = infer_frequency(
-        X["dates"],
-        debug=True,
-        window_length=WINDOW_LENGTH,
-        threshold=THRESHOLD,
-    )
-
-    assert datetime_format_check.validate(X, y) == [
-        DataCheckError(
-            message="Column 'dates' has datetime values missing between start and end date.",
-            data_check_name=datetime_format_check_name,
-            message_code=DataCheckMessageCode.DATETIME_IS_MISSING_VALUES,
-        ).to_dict(),
-        DataCheckError(
-            message="Column 'dates' has datetime values that do not align with the inferred frequency.",
-            data_check_name=datetime_format_check_name,
-            message_code=DataCheckMessageCode.DATETIME_HAS_MISALIGNED_VALUES,
-        ).to_dict(),
-        get_uneven_error("dates", ww_payload),
-    ]
-
```

Comment:
Why was this deleted?

Reply:
It looks like an exact copy of line 323 - 349